core/metacling/src/CMakeLists.txt

```diff
@@ -104,6 +104,9 @@ ROOT_LINKER_LIBRARY(Cling
         $<TARGET_OBJECTS:MetaCling>
         LIBRARIES ${CLING_LIBRARIES} ${LINK_LIBS} ${CLING_PLUGIN_LINK_LIBS})
 
+# When these two link at the same time, they can exhaust the RAM on many machines
```

Comment:
We can probably address that issue by something similar to this https://github.com/llvm/llvm-project/blob/97c571abbcea2b540511ae5a874da05bf77e5e5d/llvm/cmake/modules/HandleLLVMOptions.cmake#L30-L37

Reply:
If I read this correctly, that's only with ninja? It doesn't hurt to do this with ninja, but I would have needed a make solution.